adapters/discord/src/types/component.ts

```diff
@@ -22,6 +22,12 @@ export enum ComponentType {
   CHANNEL_SELECT = 8
 }
 
+/** https://discord.com/developers/docs/interactions/message-components#action-rows */
+export interface ActionRow {
+  type: ComponentType.ACTION_ROW,
```

Comment:
```suggestion
  type: ComponentType.ACTION_ROW
```

Useless comma (idk if there are more)